### DISCUSSION TITLE
Clarification on advertisements and tracking on Tweakers

### DISCUSSION THREAD

**Discussion by woutf** [EasyDutch-uBO/EasyDutch#37] [General]
Hi,

I just wanted to clarify something after reading [this comment ](https://github.com/EasyDutch-uBO/EasyDutch/issues/34#issuecomment-1433246721) by @JohnyP36, in which he says that Tweakers tracks users through the domain we use to serve ads. That stings because for the past two years, we've put a lot of effort into removing all third-party tracking from our website. We strongly believe in our users' rights to privacy and collecting as little data as possible. Changes we've made over the past year include getting rid of Google Analytics and moving over to server-side analytics, and switching to a different ad network that serves ads without any form of tracking. On top of that, we have a very conservative policy when it comes to types of advertising, so you won't find homepage takeovers or auto-playing video's on Tweakers. 

We can confidently say that if your main reason for blocking ads is privacy, you have nothing to worry about on Tweakers. If you just don't like ads...well...there's nothing we can do about that. They're an essential part of our business model and we try our hardest to process and display them in a responsible manner. 

If you're interested, we would be happy to share more technical details on our ad-stack and how we handle data. 

Regards,
Wout Funnekotter
Editor in Chief @ Tweakers

**Comment by Nomes77** [comment]
Hi @woutf,

Thanks for the explanation.
Although, serving ads and tracking via a first-party server is way better then via third-party, 
we as filterlist maintainers in general don't like ads and tracking at all; it doesn't matter how you serve them or how you track users.
From a business point of view I understand, but from a filterlist maintainers' point of view not.

Regards, 
EasyDutch Maintainers

**Comment by JohnyP36** [comment]
In addition to @Nomes77's statement above. We are not Adblock Plus who allows some ads & tracking. 
Users of our filterlist should make the decision themselves if they want to allow or block ads and tracking. If they want to block it, we give them rules to do. If they want to see ads, they can whitelist tweakers in the adblocker they're using. 

Thus, it's not us who worry. But the people in general and as filterlist maintainers we want our users to trust us in our job (namely blocking ads). 
Also see [@ gorhill's definition of an adblocker](https://github.com/gorhill/uBlock#general-information) and some issue in their issue tracker: [issue 1](https://github.com/uBlockOrigin/uAssets/issues/8986) & [issue 2](https://github.com/uBlockOrigin/uAssets/issues/8994). 

###### As were are not Adblock Plus we will not allow ads.

**Comment by Chrovax** [comment]
https://tweakers.net/plan/4126/tweakers-stopt-met-trackingvrije-advertenties.html

It was a noble cause, but it turns out that tracking free ads are no good for $$$ companies. Moreover, I also believe that filterlists should filter out ALL possible ads and not merely a selection: there's always a political/funding/personal motive behind it.
